Project/Sources/Classes/TestBase32.4dm

```diff
@@ -6,10 +6,13 @@ Class constructor
 	This:C1470.base32:=cs:C1710.Base32.new()
 	
 Function test_encode
+	// https://datatracker.ietf.org/doc/html/rfc4648#page-12
 	This:C1470.assertEquals(""; This:C1470.base32.encodeText(""))
-	This:C1470.assertEquals("OQ"; This:C1470.base32.encodeText("t"))
```

Comment:
Could we keep the previous tests (but could add more)

Reply:
Yes we can keep the previous test cases IF they are proven (by another tool, etc...), hence specify the origin/source of test data :-)

This is why for crypto, encoding, etc... I tend to look for official "test vectors".

It is tempting to fill test cases with the current result of the function. Then, the test case will only prove the function has not "regressed" but if the function is wrong in the first place, it does not help.

And as for OTP, I would (YAKAFOKON) do a pure official base32 class that conforms to the RFC and "extend" from that by removing the "=" in the OPT usage of base32.
What do you think ?

Thanks anyway...

:-)

Reply:
Take from RFC is a good practice. I am able to create mine to demonstrate the bug, and did not take time to fix it (I do use this OTP lib yet, do just for fun, so I expect someone that use it will fix  it, so good timing)

I make my test using OTP lib used by many user, check with base32 php and JavaScript lib and also online

--

Yes a better class could be done, see last comment. I agree with pure Base32 first

We could fix here, or share in discuss or copy paste code to make your own repository about base32 (no license issue) 
[Because here it's just for OTP]